translations/base-es.yaml

```diff
@@ -708,21 +707,19 @@ settings:
         compactBuildingInfo:
             title: Información Compacta de Edificios
             description: >-
-                Acorta la caja de información mostrando solo sus ratios. Si no, se mostrara una descripción y una imagen.
+                Acorta la caja de información mostrando solo sus ratios. Si no, se mostrará una descripción y una imagen.
         disableCutDeleteWarnings:
             title: Deshabilitar las advertencias de Cortar/Eliminar
             description: >-
-                Deshabilita los dialogos de advertencia que se muestran cuando se cortan/eliminan mas de 100 elementos.
+                Deshabilita los diálogos de advertencia que se muestran cuando se cortan/eliminan mas de 100 elementos.
 
         enableColorBlindHelper:
-            title: Modo para Daltonicos
-            description: Activa varias herramientas que permiten jugar si eres daltonico.
+            title: Modo para Daltonismo
+            description: Activa varias herramientas que permiten jugar si tienes alguna forma de daltonismo
         rotationByBuilding:
-          title: Rotation by building type
+          title: Rotación por tipo de edificio
           description: >-
-            Each building type remembers the rotation you last set it to individually.
-            This may be more comfortable if you frequently switch between placing
-            different building types.
+            Cada tipo de edificio recuerda la última rotación que seleccionaste individualmente. Esto puedo se mucho más cómodo si sueles cambiar a menudo entre colocar diferentes tipos de edificio.
```

Comment:
there are a mistake, is: "Esto puede ser mucho más cómodo si cambias a menudo entre colocar diferentes tipos de edificio."

"puede" with "E"
"ser" ended in r
"sueles" and "a menudo" is redundant